components/overflow-group/overflow-group.js

```diff
@@ -0,0 +1,495 @@
+			:host([dir="rtl"]) .d2l-overflow-group-container ::slotted(.d2l-overflow-group-custom-item) {
+				margin-left: 0.6rem;
+				margin-right: 0;
+			}
+
+			:host([opener-style="subtle"]) .d2l-overflow-group-container ::slotted(d2l-button-subtle) {
+				margin-right: 0.2rem;
+			}
+
+			:host([opener-style="subtle"][dir="rtl"]) .d2l-overflow-group-container ::slotted(d2l-button-subtle) {
+				margin-left: 0.2rem;
+				margin-right: 0;
+			}
+
+			:host([opener-style="subtle"]) .d2l-dropdown-subtle-opener-text,
+			.d2l-dropdown-opener-text {
+				margin-right: 0.3rem;
+				vertical-align: middle;
+			}
+			:host([dir="rtl"]) .d2l-dropdown-opener-text,
+			:host([opener-style="subtle"]:dir(rtl)) .d2l-dropdown-subtle-opener-text {
+				margin-left: 0.3rem;
+				margin-right: 0;
+			}
+			:host .d2l-overflow-group-container ::slotted([data-is-chomped]) {
+				display: none !important;
+			}`
+		];
+	}
+	constructor() {
+		super();
+		this._handleResize = this._handleResize.bind(this);
+		this._addEventListeners = this._addEventListeners.bind(this);
+		this._chomp = this._chomp.bind(this);
+		this._getLayoutItems = this._getLayoutItems.bind(this);
+		this._handleSlotChange = this._handleSlotChange.bind(this);
+		this._getOverflowMenu = this._getOverflowMenu.bind(this);
+
+		this._throttledResize = throttle(this._handleResize, 15);
+
+		this._overflowHidden = this._overflowHidden ?? false;
+		this.autoShow = this.autoShow ?? false;
+		this.maxToShow = this.maxToShow ?? -1;
+		this.minToShow = this.minToShow ?? 1;
+		this.openerStyle = this.minToShow ?? OPENER_STYLE.DEFAULT;
+		this._mini = this.openerType === OPENER_TYPE.ICON;
+
+		this._refId = 0;
+		this._slotItems = [];
+		this._overflowItems = [];
+	}
+	connectedCallback() {
+		super.connectedCallback();
+		this._addEventListeners();
+	}
+	disconnectedCallback() {
+		super.disconnectedCallback();
+		this._removeEventListeners();
+	}
+	async firstUpdated() {
+		super.firstUpdated();
+
+		// selected elements
+		this._buttonSlot = this.shadowRoot.querySelector('.item-container');
+		this._buttonSlot.addEventListener('slotchange', this._handleSlotChange);
+
+		this._container = this.shadowRoot.querySelector('.d2l-overflow-group-container');
+
+		this._autoShowClass = 'd2l-overflow-group-show';
+		this._autoNoShowClass = 'd2l-overflow-group-no-show';
+
+		// get the items from the button slot
+		this._slotItems = this._getSlotItems();
+		// convert them to layout items (calculate widths)
+		this._layoutItems = this._getLayoutItems(this._slotItems);
+		// convert to dropdown items (for overflow menu)
+		this._dropdownItems = this._slotItems.map((node) => this._convertToDropdownItem(node));
+
+		if (this.autoShow) {
+			this._autoDetectBoundaries(this._slotItems);
+		}
+
+		this._availableWidth = this._container.clientWidth;
+	}
+	render() {
+		let overflowMenu;
+		if (!this._overflowMenuHidden) {
+			overflowMenu = this._getOverflowMenu();
+		}
```

Comment:
```suggestion
		const overflowMenu = !this._overflowMenuHidden ? this._getOverflowMenu() : null;
```

Or alternatively, `_getOverflowMenu()` could just return `null` if `_overflowMenuHidden` is `true`.